packages/components/src/filters/advanced/select-filter.js

```diff
@@ -64,16 +65,20 @@ class SelectFilter extends Component {
 	}
 
 	render() {
-		const { config, filter, onFilterChange, isEnglish } = this.props;
+		const { className, config, filter, onFilterChange, isEnglish } = this.props;
 		const { options } = this.state;
 		const { key, rule, value } = filter;
 		const { labels, rules } = config;
 		const children = interpolateComponents( {
-			mixedString: labels.title,
+			mixedString: sprintf(
+				'{{title}}%s{{/title}} {{rule /}} {{filter /}}',
```

Comment:
This change causes `SelectControl` type filters with no `rules` to render full width:

![screen shot 2019-02-14 at 8 57 42 am](https://user-images.githubusercontent.com/63922/52799216-a1a7b280-3036-11e9-9f6f-5b18152c0160.png)

Perhaps we should check for `config.rules` and conditionally include `{{rule /}}` in the `mixedString`?

Reply:
Thanks @jeffstieler, nice find! Updated, should be good now :)